docs/Readme.md

```diff
@@ -27,24 +29,31 @@
     [Heuristic](Learning-Environment-Design-Heuristic-Brains.md),
     [Learning](Learning-Environment-Design-Learning-Brains.md)
 * [Learning Environment Best Practices](Learning-Environment-Best-Practices.md)
-* [Using the Monitor](Feature-Monitor.md)
-* [Using the Video Recorder](https://github.com/Unity-Technologies/video-recorder)
-* [Using an Executable Environment](Learning-Environment-Executable.md)
-* [Creating Custom Protobuf Messages](Creating-Custom-Protobuf-Messages.md)
+
+Optional for first time users
+  * [Using the Monitor](Feature-Monitor.md)
+  * [Using the Video Recorder](https://github.com/Unity-Technologies/video-recorder)
+  * [Using an Executable Environment](Learning-Environment-Executable.md)
+  * [Creating Custom Protobuf Messages](Creating-Custom-Protobuf-Messages.md)
 
 ## Training
 
 * [Training ML-Agents](Training-ML-Agents.md)
+* [Using TensorBoard to Observe Training](Using-Tensorboard.md)
 * [Training with Proximal Policy Optimization](Training-PPO.md)
+
+### Advanced Training Methods
```

Comment:
Shouldn't SAC be part of `Training`? @ervteng - in the SAC doc we make no mention of when to use versus PPO. We should at a minimum provide a guide of SAC vs PPO. As such, I think both PPO and SAC should be at the same level as two training methods. Thoughts?

Reply:
moving SAC to the same level as PPO

Reply:
@mmattar it seems like SAC should already be on the same level?

For the SAC vs. PPO I added a blurb to the top of both the `Training-PPO.md` and `Training-SAC.md` that compares the two. I had a discussion with @unityjeffrey during the PR and I think the plan was to eventually make an RL landing page similar to the Imitation Learning page that links to both (and others) and says "you should use X if"

Reply:
we need to create a RL training guide similar to the IL guide.  We have it in the backlog for docs, but haven't quite gotten to it.

Reply:
Sounds good.